backend/src/eventstore/eventstore.ts

```diff
@@ -15,7 +14,9 @@ export class EventStore {
     event: any;
   }>(null);
 
-  private readonly eventMap = new Map([['ItemCreatedEvent', ItemCreatedEvent]]);
```

Comment:
Nicht PR related, aber das Problem, das sich hier schon abzeichnet sind Aenderungen an Namen/API nach dem Deployen...ich nehms mal in die Problemliste im Wiki auf